packages/saved-views-client/src/models/savedViews/View.ts

```diff
@@ -19,6 +19,8 @@ export interface SavedView {
   id: string;
   displayName: string;
   shared: boolean;
+  creationTime: string;
+  lastModified: string;
```

Comment:
Could you document the format of these so that consumers can cast them to Date objects?

Reply:
Good call, updated!